### PR TITLE
Enhance libertyPackage task

### PR DESCRIPTION
**Pull request by josephca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +101,6 @@ The plugin will have made the following tasks available to your project:
 | libertyJavaDump | Dumps diagnostic information from the Liberty Profile server JVM. | 
 | libertyDebug | Runs the Liberty Profile server in the console foreground after a debugger connects to the debug port (default: 7777). | 
 | libertyStatus | Checks the WebSphere Liberty Profile server is running. |
-| libertyPackage | Generates a WebSphere Liberty Profile server archive. |
```
Comment (patricktiu) [inline README.md]: Should use `WebSphere Liberty server` in the entire document.

Comment (patricktiu) [inline README.md]: Consider splitting out into separate documents like Liberty Maven plug-in and give gradle.build sniplet example.

Comment (josephca) [inline README.md]: Fixed.

Comment (josephca) [inline README.md]: Splitted docs.

**Comment by patricktiu** [comment]
I don't see the the libertyPackage task doc.  i.e. `docs/libertyPackage.md`

**Comment by dnwarnock** [comment]
What's the plan for testing this?

**Comment by josephca** [comment]
Will add some tests for new functions.

**Comment by josephca** [comment]
More granular PR's will be created instead. Closing it now.

**Pull request closed without merging**